jenkins/ci.suse.de/templates/cloud-mkcloud-job-magnum-template.yaml

```diff
@@ -26,3 +26,8 @@
             mkcloudtarget=all
             label={label}
             job_name=cloud-mkcloud{version}-job-magnum-{arch}
+            want_barbican_proposal=0
+            want_ceilometer_proposal=0
+            want_manila_proposal=0
+            want_trove_proposal=0
+            want_sahara_proposal=0
```

Comment:
All these parameters do not exist in the job openstack-mkcloud. The list of parameters would be endless if we supported all of them -  thats not feasible.

What about using a crowbar-batch file instead? Thats a lot more flexible and you get exactly the configuraiton you want.